File: video/v1/recording_settings/create-upload-encrypted/create-upload-encrypted.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
// DANGER! This is insecure. See http://twil.io/secure
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$recording_settings = $twilio->video->v1->recordingSettings()
                                        ->create("Upload encrypted", // friendlyName
                                                 array(
                                                     "encryptionEnabled" => True,
                                                     "encryptionKeySid" => "CRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
                                                 )
                                        );

print($recording_settings->friendlyName);

File: video/v1/recording_settings/create-upload-encrypted/create-upload-encrypted.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

recording_settings = @client.video
  .recording_settings
  .create(
     encryption_enabled: true,
     encryption_key_sid: 'CRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
     friendly_name: 'Upload encrypted'
   )

puts recording_settings.friendly_name
